cmd/ak/cmd/orgs/add_member.go

```diff
@@ -7,10 +7,13 @@ import (
 
 	"go.autokitteh.dev/autokitteh/cmd/ak/common"
 	"go.autokitteh.dev/autokitteh/internal/resolver"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
+var status string
+
 var addMemberCmd = common.StandardCommand(&cobra.Command{
-	Use:     "add-member <org id> <user id>",
+	Use:     "add-member <org id> <user id> [--status status]",
```

Comment:
We don't want to allow users to add someone with the `declined` status, so I suggest you replace this with a boolean flag: `--invite` (true = `invited`, false = `active`) or `--force` (reversed true/false meaning).

BTW, that's the intended default status if not specified? Document it in the CLI.

Reply:
done